src/frontend/apps/drive/src/features/explorer/components/Explorer.scss

```diff
@@ -245,6 +380,96 @@
         gap: 8px;
         align-items: center;
       }
+
+      @media (max-width: $tablet) {
+        .explorer__search__button {
+          display: none;
+        }
+
+        .explorer__tree__actions__left {
+          flex-grow: 1;
+
+          .c__dropdown-menu-trigger {
+            flex-grow: 1;
+
+            button {
+              flex-grow: 1;
+              justify-content: center;
+            }
+          }
+        }
+      }
+    }
+
+    .c__tree-view--container {
+      @media (max-width: $tablet) {
+        display: none;
+      }
+    }
+
+    &__mobile {
```

Comment:
I know it's not the case yet, but I think it's a mistake, we have an ExplorerTree.scss, it's a shame to put all that in the Explorer.scss

Reply:
wow much violence 🤣